setup.py

```diff
@@ -158,6 +155,7 @@ def run(self):
 
 setup(
     install_requires=REQUIREMENTS,
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
```

Comment:
Adding this keyword argument will requires a quite recent version of setuptools, which might not be a available on enterprise GNU/Linux distributions. Thus this should not be used.

Reply:
If you have old setuptools, I believe this will be ignored, where it makes no difference.

If you have a recent setuptools, then this is useful and will help people install the correct version.

Shall I leave it or remove it?

Reply:
I've left it in for now, can remove it if necessary.

Reply:
I verified: Old setuptools will issue a warning, which is okay.